spec/integration/type/file_spec.rb

```diff
@@ -1528,4 +1528,134 @@ def pretty_transaction_error(transaction)
 
     raise "Got #{status_failures.length} failure(s) while applying: #{status_fail_msg}"
   end
+
+  describe "copying a file that is a link to a file", :if => Puppet.features.manages_symlinks? do
+    let(:target) { tmpfile('target') }
+    let(:link) { tmpfile('link') }
+    let(:copy) { tmpfile('copy') }
+    it "should copy the target of the link if :links => follow" do
```

Comment:
in general, ruby likes strings without interpolation in single quotes, but we don't really enforce that here, so i'm cool with it.